six/modules/c++/six.sicd/include/six/sicd/GeoData.h

```diff
@@ -27,12 +27,13 @@
 #include "six/Parameter.h"
 #include "six/ParameterCollection.h"
 #include <mem/ScopedCloneablePtr.h>
+#include <logging/Logger.h>
 
 namespace six
 {
 namespace sicd
 {
-
+struct ImageData;
```

Comment:
Throughout, include the needed headers rather than forward declaring.  That way, when someone includes this header, they don't need to also include other ones.